#1/crunch

```diff
@@ -0,0 +1,32 @@
+# Crunch
+
+## Former Participation in Celo Hackathons
+
+- Gitcoin hackathons
+
+## Goal for this Proof of Ship
+
+TBA
+
```

Comment:
Hi @lyledavids, 

this contest is focused on you having a clear goal of what you want to ship. Please fill out the details of this form.